### PR TITLE
Remove UPGRADE-5.4.md

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: UPGRADE-6.0.md [hunk 1]
```diff
@@ -173,6 +173,7 @@ Messenger
  * Removed the `prefetch_count` parameter in the AMQP bridge.
  * Removed the use of TLS option for Redis Bridge, use `rediss://127.0.0.1` instead of `redis://127.0.0.1?tls=1`
  * The `delete_after_ack` config option of the Redis transport now defaults to `true`
+ * The `reset_on_message` config option now defaults to `true`
```
Comment (fancyweb) [inline UPGRADE-6.0.md]: This line is in branch 5.4 but not in 6.0, both files should logically be the same.

Comment (fancyweb) [reply UPGRADE-6.0.md]: Oh probably because it's a FWB config and not related to the Messenger component? So it should be removed in 5.4 instead?

Comment (fabpot) [reply UPGRADE-6.0.md]: see #43203

Comment (fancyweb) [reply UPGRADE-6.0.md]: It needs to be removed on 5.4 then.

Comment (fancyweb) [reply UPGRADE-6.0.md]: Done in https://github.com/symfony/symfony/pull/44496

**Review by chalasr — APPROVED** [review]

**Review by ogizanagi — APPROVED** [review]

**Comment by ogizanagi** [comment]
Thanks @fancyweb.

**Pull request merged**